jobmaster/cvsJob/cvsJobMaster.go

```diff
@@ -26,6 +26,8 @@ import (
 	"go.uber.org/atomic"
 )
 
+var _ lib.JobMasterImpl = &JobMaster{}
```

Comment:
why need this var ?

Reply:
This is for testing purpose. We can move it somewhere else